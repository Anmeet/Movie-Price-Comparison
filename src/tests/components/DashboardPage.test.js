import React from 'react';
import DashboardPage from '../../components/Dashboard';

import { shallow } from 'enzyme';

test('should render Dashboard Page correctly', () => {
  const wrapper = shallow(<DashboardPage />);
  expect(wrapper).toMatchSnapshot();
});

test('should render the Page Title Correctly', () => {
  const wrapper = shallow(<DashboardPage />);
  expect(wrapper.find('h1').text()).toBe(" Prince's Theartre ");
});
