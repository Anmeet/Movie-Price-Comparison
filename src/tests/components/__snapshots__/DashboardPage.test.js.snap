// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Dashboard Page correctly 1`] = `
<div
  className="page-header"
>
  <div
    className="content-container"
  >
    <h1
      className="page-header__title"
    >
       Prince's Theartre 
    </h1>
    <h2
      className="page-header__subtitle"
    >
      Classic Movies At Home
    </h2>
    <p
      className="page-header__paragraph"
    >
      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation
    </p>
    <div
      className="page-body"
    />
  </div>
</div>
`;
